_data/companies.yml

```diff
@@ -13,6 +13,11 @@
   image_url: images/logo_bbm.png
   description: Operated by KMK Online as a stand-alone business with teams operating from Indonesia, Singapore, Canada, UAE and South Africa.
 
+- name: Binar Academy
+  url: https://www.binar.co.id
+  image_url: images/logo_binar.png
```

Comment:
Could you optimize the logo size to around 30 kB (currently 50 kB)? (You can use https://imageoptim.com or other tools).